insights/parsers/nsswitch_conf.py

```diff
@@ -0,0 +1,70 @@
+"""
+NSSwitchConf - file ``/etc/nsswitch.conf``
+==========================================
+
+"""
+
+from insights import LegacyItemAccess, Parser, parser
+from insights.parsers import get_active_lines
+
+
+@parser('nsswitch.conf')
+class NSSwitchConf(Parser, LegacyItemAccess):
+    """
+    Read the contents of the ``/etc/nsswitch.conf`` file.
+
+    Each non-commented line is split into the service and its sources.  The
+    sources (e.g. 'files sss') are stored as is, as a string.
+
+    nsswitch.conf is case insensitive.  This means that both the service and
+    its sources are converted to lower case and searches should be done
+    using lower case text.
+
+    Attributes:
+        data (dict): The service dictionary
+        errors (list): Non-blank lines which don't contain a ':'
+        sources (set): An unordered set of the sources seen in this file
+
+    Sample content:
```

Comment:
@PaulWay I believe you need a double colon here, `::` to keep the content formated.